src/stores/Entities/hooks.ts

```diff
@@ -43,8 +45,27 @@ export const useEntitiesStore_atLeastOneAdminTenant = () => {
     );
 };
 
-export const useEntitiesStore_capabilities_adminable = () => {
-    return useEntitiesStore(useShallow((state) => state.capabilities.admin));
+export const useEntitiesStore_capabilities_adminable = (
+    restrictByStorageMappings?: boolean
+) => {
+    const hasSupportRole = useUserInfoSummaryStore(
+        (state) => state.hasSupportAccess
+    );
+
+    return useEntitiesStore(
+        useShallow((state) => {
+            if (!restrictByStorageMappings || hasSupportRole) {
+                return state.capabilities.admin;
+            }
+
+            return Object.keys(state.storageMappings).filter(
+                (storageMappingPrefix) =>
+                    state.capabilities.admin.some((adminPrefix) =>
+                        storageMappingPrefix.startsWith(adminPrefix)
+                    )
+            );
```

Comment:
On the next update to storage mappings we should make it so this is not checked all the time.